schedule/store.go

```diff
@@ -115,6 +129,63 @@ func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, err
 
 	return result, nil
 }
+
+func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, error) {
+	return store.FindManyTx(ctx, nil, ids)
+}
+
+func (store *Store) FindManyByAssignments(ctx context.Context, tx *sql.Tx, userID string, rotationIDs []string) ([]Schedule, error) {
+	err := permission.LimitCheckAny(ctx, permission.All)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validate.UUID("UserID", userID)
+	if err != nil {
+		return nil, err
+	}
+	uid, err := uuid.Parse(userID)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	uid, err := validate.ParseUUID("UserID", userID)
	if err != nil {
		return nil, err
	}
```

Alternatively, we could just require the `userID` to be `uuid.UUID` instead of string and parse it in the GraphQL layer